File: app/src/test/java/com/cargopull/executor_driver/entity/ExecutorBalanceTest.java
package com.cargopull.executor_driver.entity;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class ExecutorBalanceTest {

  @Test
  public void testConstructor() {
    ExecutorBalance cancelOrderReason = new ExecutorBalance(1, 2, 3);
    assertEquals(cancelOrderReason.getMainAccount(), 1);
    assertEquals(cancelOrderReason.getBonusAccount(), 2);
    assertEquals(cancelOrderReason.getCashlessAccount(), 3);
  }
}

File: app/src/test/java/com/cargopull/executor_driver/entity/ExecutorBalanceTest.kt
package com.cargopull.executor_driver.entity

import org.junit.Assert.assertEquals

import org.junit.Test

class ExecutorBalanceTest {

    @Test
    fun testConstructor() {
        val cancelOrderReason = ExecutorBalance(1, 2, 3)
        assertEquals(cancelOrderReason.mainAccount, 1)
        assertEquals(cancelOrderReason.bonusAccount, 2)
        assertEquals(cancelOrderReason.cashlessAccount, 3)
        assertEquals(cancelOrderReason.summary, 6)
    }
}